File: FirstFactorial.java
import java.util.*; 
import java.io.*;

class Main {

  public static int FirstFactorial(int num) {
    // code goes here  
    int sonuc = 1;
    for(int i=num;i>0;i--){
      sonuc =sonuc * i;
    }
    
    return sonuc;
  }

  public static void main (String[] args) {  
    // keep this function call here     
    Scanner s = new Scanner(System.in);
    System.out.print(FirstFactorial(s.nextLine())); 
  }

}

File: FirstFactorial.py
# -*- coding: utf-8 -*-
"""
Created on Sun Jun 18 08:35:48 2023

@author: pc
"""

def FirstFactorial(num):

  sonuc = 1
  for a in range (1,num+1):
    sonuc = sonuc * a
  # code goes here
  return sonuc

# keep this function call here 
print(FirstFactorial(input()))